source/includes/language-compatibility-table-go.rst

```diff
@@ -1 +1,34 @@
-The MongoDB Go Driver requires Go 1.10 or later.
+The following compatibility table specifies the minimum version of the
+of the Go programming language to use with versions of the {+driver-long+}.
+
+The first column lists the driver version.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Go Driver Version
+     - Minimum Go Language Version
+   * - 1.10
+     - 1.13
+   * - 1.9
+     - 1.10
+   * - 1.8
+     - 1.10
+   * - 1.7
+     - 1.10
+   * - 1.6
+     - 1.10
+   * - 1.5
+     - 1.10
+   * - 1.4
+     - 1.10
+   * - 1.3
+     - 1.10
+   * - 1.2
+     - 1.10
+   * - 1.1
+     - 1.10
+   * - 1.0
+     - 1.10
```

Comment:
Why did you choose to go with versions in the right column instead of checkmarks? I think this presentation might be a little confusing, for example, if users see this table and at first glance think they can only use Golang v1.10 for Driver v1.9. Since the version numbers are also similar between Golang and the driver, it might also reduce confusion to have a matrix.

Reply:
Discussed sync, agreed it this makes sense as `Go` only has a minimum language version, where as other drivers can have minimum and maximum language versions.